bc125csv/scanner.py

```diff
@@ -267,29 +265,26 @@ class DeviceLookup(object): # pragma: no cover
     """
 
     def __init__(self):
-        self.context = pyudev.Context()
-
-    def is_scanner(self, device):
-        """Given USB device is a compatible scanner."""
-        return device.get("ID_VENDOR_ID") == "1965" and \
-            device.get("ID_MODEL") in SUPPORTED_MODELS
+        self.device = self._search_devices()
 
-    def is_tty(self, device):
-        """Given USB device is a serial tty."""
-        return device.get("SUBSYSTEM") == "tty"
+    def _search_devices(self):
+        """
+        Find compatible scanner and return usb device. Returns arguments for Serial constructor.
+        """
+        ports = comports(include_links=False)
+        for port in ports:
+            # Uniden Vendor
+            if port.vid == 6501 and port.product in SUPPORTED_MODELS:
+                return {
+                    "port": port.device,
+                    "baudrate": 9600
+                }
+            # Silicon Laboratories Vendor (SR30C uses this chipset for its USB controller):
+            if port.vid == 4292 and port.pid == 60000:
```

Comment:
Could we use `port.product` here as well? `port.product == "SR30C"` would be more readable in my opinion. Since I don't have an SR30C I can't check the value of `port.product` for this case.